internal/job/restore/restore.go

```diff
@@ -143,12 +145,12 @@ func (r *Restore) doRestoreRawImagePhase(privateData *restorePrivateData, raw *j
 	return setRestorePrivateData(r.repo, r.actionUID, privateData)
 }
 
-func (r *Restore) loopCopyChunk(privateData *restorePrivateData, rawImageSize uint64) error {
-	chunkCount := int(math.Ceil(float64(rawImageSize) / float64(r.rawImageChunkSize)))
+func (r *Restore) loopCopyChunk(privateData *restorePrivateData, rawImageSize uint64, rawChecksumChunkSize uint64, enableChecksumVerify bool) error {
+	chunkCount := int(math.Ceil(float64(rawImageSize) / float64(rawChecksumChunkSize)))
```

Comment:
I'm not too sure `rawChecksumChunkSize` is suitable here. We need to count the data chunks, so `r.rawImageChunkSize` seems reasonable.